pkgs/development/python-modules/async-tkinter-loop/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, pkgs
+, python3
+, buildPythonPackage
+, fetchPypi
+, poetry-core
+, pythonRelaxDepsHook
+}:
+buildPythonPackage rec {
```

```suggestion
}:

buildPythonPackage rec {
```
insert a blank line,